Library/Formula/gneural-network.rb

```diff
@@ -0,0 +1,24 @@
+class GneuralNetwork < Formula
+  desc "GNU Gneural Network implements a programmable neural network."
```

Comment:
No need to have the name of the software repeated in the description.  Just something like "Programmable neural network".